src/fsharp/TypedTreeOps.fs

```diff
@@ -1393,11 +1393,13 @@ type ValHash<'T> =
         let (ValHash t) = ht
         t.Values :> seq<'T>
 
-    member ht.TryFind (v: Val) = 
+    member ht.TryGetValue(v:Val, [<System.Runtime.InteropServices.Out>] value: byref<'T>) = 
```

Comment:
Is this attribute required, or is just the byref sufficient?

Reply:
I am not totally sure tbh. I copied it from other places. AFAIK it was required earlier

Reply:
This should be `TryGetValue(v: Val, value: outref<'T>)`